modules/ROOT/pages/setup-vmbaremetal-cluster.adoc

```diff
@@ -40,7 +41,8 @@ To standardize your own bare-metal solutions using public Kubernetes distributio
 ** https://www.redhat.com/en/technologies/cloud-computing/openshift/container-platform[RedHat OpenShift Container Platform (OCP)^]
 ** https://www.redhat.com/en/technologies/cloud-computing/openshift/platform-plus[RedHat OpenShift Platform Plus^]
 
-If you don’t have investments in Kubernetes and must stay in your own hosted environment, https://www.google.com/url?q=https://www.mulesoft.com/lp/dl/mule-esb-enterprise&sa=D&source=docs&ust=1680125428223709&usg=AOvVaw3DzPT38SO_tIf8PhG-wDtd[Mule runtime standalone^] is an available option. This deployment model has reduced runtime resiliency and heavy management overhead.
+[start=4]
+. If you don’t have investments in Kubernetes and must stay in your own hosted environment, https://www.google.com/url?q=https://www.mulesoft.com/lp/dl/mule-esb-enterprise&sa=D&source=docs&ust=1680125428223709&usg=AOvVaw3DzPT38SO_tIf8PhG-wDtd[Mule runtime standalone^] is an available option. This deployment model has reduced runtime resiliency and heavy management overhead.
```

Comment:
Is this link correct? What about removing the google redirect? Or just linking to the Mule standalone docs?

Reply:
Or maybe also give the link to the docs?